sdk/textanalytics/Azure.AI.TextAnalytics/tests/MockTests/Legacy/TextAnalyticsClientMockTests.cs

```diff
@@ -11,8 +11,9 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Azure.AI.TextAnalytics.Tests
+namespace Azure.AI.TextAnalytics.Legacy.Tests
```

Comment:
Interesting that we've got tests dedicated to legacy only.   I'd like to better understand the strategy there.   In theory, we should be able to test all service versions with the same test, no?

Reply:
This happens only for the Mock tests. The reason why I wanted to separate these tests was that the input and output from the service are slightly different so the "output mock" is affected.